src/model/OWL.plt

```diff
@@ -0,0 +1,172 @@
+:- begin_tripledb_tests(
+        'model_OWL',
+        'package://knowrob/owl/test/test_owl.owl',
+        [ namespace('http://knowrob.org/kb/test_owl#')]
+  ).
+
+:- use_module('./OWL.pl').
+
+:- use_module(library('lang/terms/is_a'),
+    [ subclass_of/2,
+      subproperty_of/2
+    ]).
+
+:- use_module(library('db/tripledb'),
+    [ tripledb_load/2,
+      tripledb_ask/3
+    ]).
+
+:- use_module(library('rostest')).
+
+test('is_class') :-
+  assert_true(is_class(test:'E')),
+  assert_true(is_class(owl:'Thing')),
+  assert_true(is_class(owl:'Nothing')),
+  assert_false(is_class(test:'DoesNotExists')),
+  assert_false(is_class(test:'p')),
+  assert_false(is_class(test:'bIndiv')),
+  assert_false(is_class(test:'P')).
+
+test('is_restriction/1') :-
+  assert_true((
```

Comment:
it is probably better to wrap each term in `assert_true` because you will get more useful output when it fails.
Maybe you can add a helper predicate that does some common checks for restrictions.